### PR TITLE
626 dark theme

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#626 

- Updates dark theme values
- Page templates (header and side menu)
- Start page with summary cards
- Pages with a filterable table
- Passing whole theme object into styles file (breaking change)
- Introduced basic styles to be reused (breaking change)

image

image

image

image

image

image

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoFailedTasksBadge/WfoFailedTasksBadge.tsx [hunk 1]
```diff
@@ -81,12 +78,14 @@ export const WfoFailedTasksBadge = () => {
...
                 <WfoHeaderBadge
-                    color={theme.colors.emptyShade}
+                    color={theme.colors.ghost}
                     textColor={theme.colors.shadow}
                     iconType={() => (
                         <WfoCheckmarkCircleFill color={theme.colors.success} />
                     )}
-                />
+                >
+                    0
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoFailedTasksBadge/WfoFailedTasksBadge.tsx:87]: Intentional default value?

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTableSettingsModal/WfoTableSettingsModal.tsx [hunk 2]
```diff
@@ -71,15 +77,14 @@ export const TableSettingsModal = <T,>({
                 })
             }
         >
+            {/* Todo not sure how to target the text in the form - it needs a better color */}
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoTable/WfoTableSettingsModal/WfoTableSettingsModal.tsx:80]: Any progress on this? Should it be a ticket?

File: packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts [hunk 3]
```diff
@@ -1,18 +1,37 @@
...
+    EuiThemeComputed,
+    shade,
+    tint,
+    useEuiTheme,
+} from '@elastic/eui';
 
-export const useOrchestratorTheme = () => {
+import { ColorModes } from '@/types';
+
+export type WfoTheme = {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts:11]: You asked for input about naming this type. I dont know a better term, WfoTheme seems fine, it's everything theme related.

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTableSettingsModal/WfoTableSettingsModal.tsx (hunk 2 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/components/WfoTable/WfoTableSettingsModal/WfoTableSettingsModal.tsx:80]: whoops accidental commit, i found it haha

File: packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts (hunk 3 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts:11]: Yeah might be too close to the `theme` prop we use everywhere. Keeping it like this for now

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/styles.ts [hunk 4]
```diff
@@ -1,18 +1,14 @@
-import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
 import { css } from '@emotion/react';
 
-import { toHexColorWithOpacity } from './utils/toHexColorWithOpacity';
```
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/styles.ts:4]: added a task in the story to remove this when it is not used after finishing the whole implementation of the dark theme

**Review by Georgi2704 — APPROVED** [review]

**Pull request merged**